apps/root/src/pages/transfer/components/recipient-address/components/contact-list-modal/index.tsx

```diff
@@ -22,7 +22,15 @@ const ContactListModal = ({ shouldShow, setShouldShow, onClickContact }: Contact
   const noContactsModalContent = React.useMemo(
     () => (
       <Grid container justifyContent="center" rowSpacing={6}>
-        <Grid item direction="column" gap={2} textAlign="center" color={colors[themeMode].typography.typo3}>
+        <Grid
```

Comment:
sadly a grid cannot be an item AND a container at the same time per MUI recommendation